Resources/Descriptives/qml/Descriptives.qml

```diff
@@ -134,8 +134,21 @@ Form
             {
                                     name: "plotVariables";			label: qsTr("Distribution plots")
                 CheckBox {			name: "distPlotDensity";		label: qsTr("Display density")					}
+				CheckBox {			name: "distPlotRug";		label: qsTr("Display rug marks")					}
+				DropDown {
+                name: "binWidthType"
+                label: qsTr("Bin Width Type")
+                indexDefaultValue: 0
+                values:
+                [	
+					{ label: qsTr("Sturges"),				value: "sturges"	},
+                    { label: qsTr("Scott"),					value: "scott"		},
+                    { label: qsTr("Doane"),					value: "doane"	},
+					{ label: qsTr("Freedman-Diaconis"),		value: "fd"	}
+                ]
+				}
             }
```

Comment:
you could add a manual options like this:
```suggestion
               values:
						[
							{ label: qsTr("Sturges"),				value: "sturges"	},
							{ label: qsTr("Scott"),					value: "scott"		},
							{ label: qsTr("Doane"),					value: "doane"	},
							{ label: qsTr("Freedman-Diaconis"),		value: "fd"	},
							{ label: qsTr("Manual"),				value: "manual"	}
						]
					id: binWidthType
				}
				DoubleField
				{
					name:			"numberOfBins"
					label:			qsTr("Number of Bins")
					defaultValue:	30
					min:			3;
					max:			10000;
					enabled:		binWidthType.currentValue === "manual"
				}
```
and then users can specify the number of bins manually.